worker/raft/worker.go

```diff
@@ -40,6 +41,14 @@ const (
 	// was. If it crashes instead the logging will give a path to the
 	// problem.
 	LoopTimeout = 1 * time.Minute
+
+	// noLeaderTimeout is how long a follower will wait for contact
+	// from the leader before restarting. This allows us to see config
+	// changes (force-appended by the raft-backstop worker) to allow
+	// us to become voting again if the leader was removed leaving a
+	// 2-node cluster without quorum.
+	noLeaderTimeout   = 1 * time.Minute
+	noLeaderFrequency = 10 * time.Second
```

Comment:
Should comment what noLeaderFrequency does. I can guess, but...

Reply:
Wilco.